tests/helpers.test.ts

```diff
@@ -1,55 +1,95 @@
     });
 })
 
 describe('cleanExpectedTranscription', () => {
-    test('Replaces hyphens with space', () => {
+    it('Replaces hyphens with space', () => {
         expect(cleanExpectedTranscription('test-harness')).toEqual('test harness');
     });
 
-    test('Lowercases strings', () => {
+    it('Lowercases strings', () => {
         expect(cleanExpectedTranscription("THIS IS SPARTA")).toStrictEqual('this is sparta');
     });
 })
 
 
 describe('createTestData', ()=>{
-    test('Throws an error when file path is undefined', () => {
+    it('Throws an error when file path is undefined', () => {
         expect(() => { createTestData('', './audio-recording-info') }).toThrowError(Error);
     });
 
-    test('Throws an error when folder path is undefined', () => {
+    it('Throws an error when folder path is undefined', () => {
         expect(() => { createTestData('./audio-recording-info/transcription.txt', '') }).toThrowError(Error);
     });
 
-    test('Throws an error when folder path is NOT pointing at a folder', () => {
+    it('Throws an error when folder path is NOT pointing at a folder', () => {
         expect(() => { createTestData('./audio-recording-info/transcription.txt', './jest.json') }).toThrowError(Error);
     });
 
-    test('Throws an error when file extension is not .txt', () => {
+    it('Throws an error when file extension is not .txt', () => {
         expect(() => { createTestData('./audio-recording-info/greatSample.mp3', '') }).toThrowError(Error);
     })
 
     describe('parseTextFileContent', () => {
     
-        const sampleFileContent = 'sample.wav\tthis is only a sample';
-        const fakeAudioFolder = './audio-recording-info'
-        const fakeItemInResult = {recording:`${fakeAudioFolder}\\sample.wav`, transcription:'this is only a sample'}
-        const results = parseTextFileContent(sampleFileContent, fakeAudioFolder)
-        test('Returns Array', () => {
-            expect(Array.isArray(results)).toBe(true)
+        it('Returns Array', () => {
+            expect(Array.isArray(testData)).toBe(true)
         })
-        test('Array Contains TestData', () => {
-            expect(results[0]).toEqual(fakeItemInResult)
+        it('Array Contains TestData', () => {
+            expect(testData[0]).toEqual(fakeItemInResult)
         });
     });
-})
+});
+
+describe('ITestResults', () => {
+    it('should track expectedTranscription', () => {
+        expect(TestResultsMock()).toHaveProperty('expectedTranscription')
+    });
+
+    it('should track actualTranscription', () => {
+        expect(TestResultsMock()).toHaveProperty('actualTranscription')
+    });
+
+    it('should track wordErrorRate', () => {
+        expect(TestResultsMock()).toHaveProperty('wordErrorRate')
+    });
+});
+
+describe('calculateSER', () => {
+    
+    const ser = calculateSERMock([TestResultsMock()])
+    it('Accept an array of ITestResult Objects',()=>{
```

Comment:
The description of this test doesn't seem to match the asserted expectation. Just because the function was called does not tell us anything about the parameters it accepts. If the purpose of this test is to check how the function handles different parameters, consider writing some "bad path" scenarios as well

Reply:
I will address this in the upcoming PR.